setup.py

```diff
@@ -35,7 +35,7 @@
     'ROK': ['requests', 'pyjwt', 'simplejson'],
     'sap_hana': ['pyhdb>=0.3.4'],
     'soap': ['zeep', 'lxml==4.2.5'],
-    'snowflake': ['snowflake-connector-python>=2.5'],
+    'snowflake': ['snowflake-connector-python>=2.5', 'pyjwt>=1.6.3'],
```

Comment:
I choose 1.6.3 because it's the first version when the `PyJWTError` exception exist: https://pyjwt.readthedocs.io/en/latest/changelog.html#v1-6-3
(and we import it in the snowflake_connector module)